kubeflow-components/overlays/moc/profiles/demo-profile.yaml

```diff
@@ -0,0 +1,15 @@
+---
+apiVersion: kubeflow.org/v1beta1
+kind: Profile
+metadata:
+    name: kf-demo
```

Comment:
This corresponds to the namespace created. Does it create the RoleBinding for the user?

Reply:
It does create a rolebinding for the user. It  gives a role called `kubeflow-admin` ([link](https://github.com/kubeflow/manifests/blob/f95406b14a2025db1e8b88d72caa1316ec5d1f49/kubeflow-roles/base/cluster-roles.yaml#L22))